Point_set_processing_3/include/CGAL/remove_outliers.h

```diff
@@ -256,7 +256,11 @@ remove_outliers(
   {
     std::nth_element (f2r,
                       sorted_points.begin() + first_index_to_remove,
-                      sorted_points.end());
+                      sorted_points.end(),
+                      [&point_map](const std::pair<FT, value_type>& v1, const std::pair<FT, value_type>& v2)
+                      {
+                        return get(point_map, v1.second) < get(point_map, v2.second);
+                      });
```

Comment:
Actually the `nth_element` is supposed to be done on the computed distance, so it should `return v1.first < v2.second`. I don't understand how I did not see that this code was obviously wrong (on master) and did not detect errors with it before…